edx-platform/pearson-vue-theme/lms/templates/header/navbar-logo-header.html

```diff
@@ -0,0 +1,61 @@
+## mako
+
+<%page expression_filter="h" args="online_help_token"/>
+
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.urls import reverse
+from django.utils.translation import gettext as _
+from lms.djangoapps.ccx.overrides import get_current_ccx
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.features.enterprise_support.utils import get_enterprise_learner_generic_name, get_enterprise_learner_portal
+
+# App that handles subdomain specific branding
+from lms.djangoapps.branding import api as branding_api
+%>
+
+<%
+enterprise_customer_link = get_enterprise_learner_portal(request)
```

Comment:
This module will fail with the new integration of enterprise. Please check if the implementation in juniper stil works.

Reply:
@anfbermudezme  I'll remove it